### ISSUE TITLE
Remove option in optionset associated with question: ua_ruhr_v1.1.0-beta.1/technical-classification/data-volume/volume

### ISSUE THREAD

**Issue by smohex** [FDM-UARuhr/rdmo-catalog-uaruhr#56]
**Describe the request**  
If all "not yet defined" are checked, the statement is not very helpful for planning. Maybe a rough / approximate / estimated would be better

**Propose what exactly should be changed**  
Delete selection "not yet defined".

**From the following list delete everything that is NOT part of the content of your proposal. Once a subject of change is approved it will be marked accordingly in the list.**  

- [ ] edit optionset "dataset_size_options"

**Comment by smohex** [comment]
the value "not yet defined" is already used in projects (test as well as productive system). What happens when the value is removed?

**Comment by jmfrenzel** [comment]
The proposed change, i.e. removing an option from the "dataset_size_options" option set, is a breaking change because it changes the data entered by the user. 
In addition such change will deviate us from [rdmorganiser/rdmo-catalog](https://github.com/rdmorganiser/rdmo-catalog) so we have to maintain this change. 

* This type of changes I would recommend to promote rdmo community (content group), but not to remove the option. Add new condition & task.
  * RUB :+1: 
  * UDE :+1: 
  * TUDO :+1:

**Comment by jmfrenzel** [comment]
https://github.com/rdmorganiser/rdmo-catalog/issues/54

**Comment by ClaraRosalie** [comment]
is being implemented in upstream https://github.com/rdmorganiser/rdmo-catalog/pull/108
